edit/edit.go

```diff
@@ -73,13 +73,20 @@ func InterpretLabelForWorkspaceLocation(root string, target string) (buildFile s
 
 	defaultBuildFileName := "BUILD"
 	if strings.HasPrefix(target, "//") {
+		pkgPath := filepath.Join(rootDir, pkg)
+		if isFile(pkgPath) {
+			// allow operation on other files like WORKSPACE
+			buildFile = pkgPath
+			pkg = filepath.Dir(pkg)
```

Comment:
Shouldn't it be `path.Dir` instead? `filepath` is OS-specific and works correctly with local paths, but packages defined by labels always have forward slashes, `path.Dir` seems to be a better solution here (even if `filepath.Dir` parses paths with forward slashes too).

Reply:
I copied this from below, but pushed a new commit that uses `path.Dir` and also explicitly uses `filepath.FromSlash` to convert logical to platform paths.